consensus/ibft/ibft.go

```diff
@@ -490,8 +482,7 @@ func (i *Ibft) runSyncState() {
 
 	for i.isState(SyncState) {
 		// try to sync with the best-suited peer
-		p := i.syncer.BestPeer()
-		if p == nil {
+		if !i.syncer.HasSyncPeer() {
```

Comment:
The naming here should be different - you're effectively looking for a peer that has a better chain than you do (bigger height) to jump out of RoundChangeState or to know if you should attempt syncing with them.

What do you think about `HigherChainExists`?

cc @dbrajovic

Reply:
`HigherChainExists` is not unclear, I think. It's hard to see where the chain is?